java/registry/src/main/java/dev/sunbirdrc/registry/dao/VertexReader.java

```diff
@@ -346,6 +348,12 @@ private ArrayNode expandChildObject(ObjectNode entityNode, List<String> processe
                                     if (ovalue != null) {
                                         resultArr.add(ovalue);
                                     } else {
+                                        if (node.isObject()) {
+                                            Map<String, Object> deletedNodeRef = JSONUtil.convertJsonNodeToMap(node);
+                                            deletedNodeRef.put(uuidPropertyName , osidVal);
+                                            JsonNode refactorNode = new ObjectMapper().convertValue(deletedNodeRef , JsonNode.class);
+                                            resultArr.add(refactorNode);
+                                        }
```

Comment:
Adding the deleted object is not expected, we should we able to return empty array instead.
```
   ...
   "educations": [
        {
            "osid": "1-df55d012-cbfe-4548-ad1c-b7fc114d1161"
        }
    ],
    ...
```

Also it is returning unexpected response after attested
```
    ...
    "educations": [
        {
            "osid": "1-"
        }
    ],
    ...
```